pkg/runners/zero/zero.go

```
@@ -210,6 +231,21 @@ func (runner *ZeroRunner) steps() uint64 {
 	return runner.vm.Step
 }
 
+func (runner *ZeroRunner) Output() []*fp.Element {
+	output := []*fp.Element{}
+	for _, segment := range runner.segments() {
+		if segment.BuiltinRunner.String() == "output" {
+			for offset := uint64(0); offset < segment.Len(); offset++ {
+				value := segment.Peek(offset)
+				valueFelt, _ := value.FieldElement()
```

Is it safe to ignore the error on line 240? Does output can only hold felts?

---

left a todo. we'll solve all this later